blocks/v2-images-grid/v2-images-grid.css

```diff
@@ -48,11 +65,163 @@
 .v2-images-grid__item:nth-child(3) img {
   aspect-ratio: 4/3;
 }
- 
+
 .v2-images-grid__figcaption {
   display: none;
 }
 
+.v2-images-grid__modal-content {
+  display: flex;
+  flex-direction: column;
+  position: relative;
+  height: auto;
+  justify-content: flex-end;
+}
+
+.v2-images-grid__carousel-items-list,
+.v2-images-grid__carousel-preview-list {
+  display: flex;
+  padding: 0;
+  list-style: none;
+  margin: 0;
+  max-width: 100%;
+  scroll-behavior: smooth;
+  scroll-snap-type: x mandatory;
+  scrollbar-width: none;
+  overflow: scroll hidden;
+  flex-wrap: nowrap;
+}
+
+.v2-images-grid__carousel-items-list::-webkit-scrollbar,
+.v2-images-grid__carousel-preview-list::-webkit-scrollbar {
+  display: none;
+}
+
+.v2-images-grid__carousel-preview-list {
+  gap: 10px;
+  display: flex;
+  list-style: none;
```

Comment:
Some of the styles are repeated above at rule 82. So these can be removed (display, list-style, margin, padding

Reply:
done